tests/utilities/encodedImageStorage/uploadEncodedImage.spec.ts

```
@@ -32,7 +33,8 @@ describe("src -> utilities -> encodedImageStorage -> uploadEncodedImage", () =>
         "NAAAAKElEQVQ4jWNgYGD4Twzu6FhFFGYYNXDUwGFpIAk2E4dHDRw1cDgaCAASFOffhEIO" +
         "3gAAAABJRU5ErkJggg==";
       await uploadEncodedImage(img, null);
-    } catch (error: any) {
+    } catch (error: unknown) {
+      if (!(error instanceof ApplicationError)) return;
```

Make the test fail here instead of returning.